src/internal/components/session/CommonSessionCard.tsx

```diff
@@ -166,7 +183,7 @@ export function CommonSessionCard({cardName, faIcon, cardInfo, children}: ICommo
 
     return <>
         <h4 className="text-left d-flex align-items-center mb-3"><i className={`mr-2 ordino-icon-2 fas ${faIcon}`} ></i>{cardName}</h4>
-        <Card ref={parent} className="shadow-sm">
+        <Card ref={parent} className={`card-shadow ${highlight ? 'highlight-card' : ''}`} onAnimationStart={onHighlightAnimationStart} onAnimationEnd={onHighlightAnimationEnd}>
```

Comment:
I've added the `onAnimationStart` and `onAnimationEnd` event listeners from React. They are fired when the CSS animation starts/ends. 

* see https://reactjs.org/docs/events.html#animation-events
* example: https://blog.bitsrc.io/how-to-trigger-animation-events-using-react-a6c3256b21c6